packtools/catalogs/default-html.xslt

```diff
@@ -44,7 +43,7 @@
             <xsl:if test="article/back/ack">
               <li><a href="#acknowledgement">Acknowledgement</a></li>
             </xsl:if>
-            <xsl:if test="article/back/ref-list">
+            <xsl:if test="article//back/ref-list">
               <li><a href="#references">References</a></li>
```

Comment:
sim, foi parte de um test para gerar uma "table of content" da página. A ideia não era de deixar isso no layout final, ou pelo menos não com conteúdo estático.